### PR TITLE
Write a roadmap, resolves #65

### LINKED ISSUE #65: Write a roadmap
We could really use the help of recording what we are working on, and what we plan or wish to work on in the future. What is better than a good ol' roadmap for the job!

@AvivSela and I started working on this in the last Monday meetup, and I will open a pull request with a ROADMAP.md file on which we could all discuss

Found this short guide to be useful https://mozillascience.github.io/working-open-workshop/roadmapping/

### PR DESCRIPTION
**Pull request by cjer**
You all are very much invited to comment and disscuss

### CODE REVIEW

**Comment by cjer** [comment]
This will resolve issue #65

**Comment by cjer** [comment]
This will resolve #65

**Comment by GipsyBeggar** [comment]
About the **Misc** clause
Which Repos do you wish to add?

**Comment by cjer** [comment]
I think all the 15 minutes complaint code definitely could go in another repo. Same goes for once we start working on a website to put our stuff in. And also maybe analytics we do, since some of it is completely unrelated to our core code (my jupyter notebooks for example, which are sitting in my private repo right now)

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+## Goals for OpenBus
+### Real-Delay - QoS Score
+*Create a QoS (Quality-of-Service) score for public transport that can be viewed and sliced by agency, route, trip, stop, date and time.*  
+*We wish to compare the planned trips to the actual trips (GTFS to SIRI). This information might help us discover problems such as trips that didn't happen, areas with bad traffic that buses usually are late for, etc.*  
+	
+### Accessibility Score
+*Show how accessible are different locations and resources by public transport.*  
+*We wish to measure accessibility between two areas, for example how many people that live in South Tel Aviv are connected to Kfar-Saba. And also to measure accessibility between area and some features like workplaces, universities, healthcare, culture etc. for example Many many people who live in Beer-Sheva are connected to workplaces?*  
+-   Between Locations
+-   Between Locations and Resources
```
Comment (amitBotzer) [inline ROADMAP.md]: What does resources mean?

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+-   Between Populations and Resources
```
Comment (amitBotzer) [inline ROADMAP.md]: What does resources mean?

File: ROADMAP.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
+# OpenBus Roadmap
+
+## Project Mission and Summary
+We use public data to improve public transport in Israel.  
+Open Bus is a project of The Public Knowledge Workshop. 
+
+## How to Get Involved
+We welcome any kind of contribution. Look for [good first issue](https://github.com/hasadna/open-bus/labels/good%20first%20issue) and [help wanted](https://github.com/hasadna/open-bus/labels/help%20wanted) labeled issues, and make sure to read our [contributing guidlines](https://github.com/hasadna/open-bus/blob/master/CONTRIBUTING.md).
```
[the remark this answers is not in the document]
Comment (cjer) [reply ROADMAP.md]: Yeah, we still have to write one 👍

File: ROADMAP.md (hunk 2 above)
Comment (cjer) [reply ROADMAP.md]: fixed

File: ROADMAP.md (hunk 1 above)
Comment (cjer) [reply ROADMAP.md]: fixed

File: ROADMAP.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+-   Between Populations and Resources
+
+### Complaints
+*Work with 15-minutes on building a good complaint platform. Use complaints as a valuable data source.*  
+
+### Website
+*A suitable space to throw our stuff in and share them with the world.*
+
+### Misc
+-   Analyze Incentives for using private car (public & private sector)
```
[the remark this answers is not in the document]
Comment (cjer) [reply ROADMAP.md]: fixed

**Pull request merged**